src/services/pcn-bridge/src/Bridge_dp_stp.c

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright 2019 The Polycube Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
```

Comment:
@mauriciovasquezbernal Does it make sense having two dataplanes? It looks the code would become more difficult to maintain. I would use conditional compilation instead of having two different dataplanes.

Reply:
I agree, not sure however how easy is to merge the two dataplanes. 
Maybe the merged version is less readable than the separated ones.

Reply:
It seems like there are only a few differences between the two files, it should not be so much hard to merge them so i could try if you want. Let me know!

Reply:
Please, it could be interesting if you give it a try.